### PR TITLE
1902 v1

### PR DESCRIPTION
**Pull request by deusexludus**
created a new xml based on 1901 - added new units.

### CODE REVIEW

File: map.yml
Comment (deusexludus) [inline map.yml:1]: error in row 5, fixed in commit 3

**Comment by beelee1** [comment]
@deusexludus Idk but maybe @RogerCooper can merge this. Otherwise you'll have to wait on dan

**Comment by RogerCooper** [comment]
I will try this over the weekend.

**Comment by RogerCooper** [comment]
It is merged.

**Comment by deusexludus** [comment]
Thanks beelee, thanks Roger!

**Pull request merged**